spring-cloud-commons/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,55 @@
+{
+  "hints": [],
+  "groups": [],
+  "properties": [
+    {
+      "sourceType": "org.springframework.cloud.autoconfigure.RefreshAutoConfiguration",
+      "defaultValue": "true",
+      "name": "spring.cloud.refresh.enabled",
+      "description": "Enables autoconfiguration for the refresh scope and associated features.",
```

Comment:
I don't know if Spring Cloud has a consistent way of documenting flags (or any property for that matter). If that's what you use, all good. 

(Just wanted to mention this as we try to make sure documentation is as much as consistent as possible).

Reply:
> I don't know if Spring Cloud has a consistent way of documenting flags (or any property for that matter)

We don't really.  Is there something inconsistent that you are seeing?

Reply:
> We don't really. Is there something inconsistent that you are seeing?

I didn't notice anything. It was really more some context on what we do rather than anything you've done in this PR. Sorry, that wasn't clear.